web/src/main/webapp/WEB-INF/data/config/schema_plugins/iso19139/index-fields.xsl

```diff
@@ -255,19 +255,33 @@
 
 				<!-- Index platforms -->
 
-				<xsl:if test="gmd:thesaurusName/*/gmd:title/*/text()='AODN Platform Vocabulary'">
+				<xsl:if test="gmd:thesaurusName/*/gmd:title/*/text()='theme.aodn_aodn-platform-vocabulary.rdf'">
```

Comment:
As discussed,  we need to keep this - GeoNetwork 3 and the the GA transform will put the correct Thesaurus title in the metadata i.e.  'AODN Platform Vocabulary'